gravitee-am-repository/gravitee-am-repository-jdbc/src/main/java/io/gravitee/am/repository/jdbc/common/dialect/PostgresqlHelper.java

```diff
@@ -131,8 +131,8 @@ protected String buildPagingClause(String field, int page, int size) {
     }
 
     @Override
-    public String buildFindUserByDomainAndEmail(ReferenceType referenceType, String referenceId, String email, boolean strict) {
-        return new StringBuilder("SELECT * FROM users u WHERE ")
+    public String buildFindUserByDomainAndEmail(ReferenceType referenceType, String referenceId, String email, boolean strict, boolean organizationUser) {
```

Comment:
Maybe you should rename this method by buildFindUserByReferenceAndEmail.

The boolean organizationUser is really needed here ? can we not use the referenceType ?